services/provider/server/server.go

```diff
@@ -810,11 +808,70 @@ func (s *OCSProviderServer) GetStorageClaimConfig(ctx context.Context, req *pb.S
 			}
 
 			// SID for RamenDR
-			storageID := calculateCephRbdStorageId(
+			storageID := calculateCephRbdStorageID(
 				cephCluster.Status.CephStatus.FSID,
 				strconv.Itoa(blockPool.Status.PoolID),
 				rns.Name)
 
+			if blockPool.GetAnnotations()[util.BlockPoolMirroringTargetIDAnnotation] != "" {
```

Comment:
If you are already loading the annotation here, you should not do it again later in the func